src/restoration-state-handler.ts

```diff
@@ -173,6 +187,10 @@ export class RestorationStateHandler
       },
     };
 
+    if (searchInside) {
+      restorationState.searchType =
+        searchInside === 'TXT' ? SearchType.FULLTEXT : undefined; // No explicit metadata sin
```

Comment:
Q: how can we get it so that when undefined, the `sin=TXT` gets removed from url?

Reply:
Should be fixed in latest commit